sdk/OoyalaSkinSDK/index.android.js

```diff
@@ -70,7 +80,12 @@ var OoyalaSkin = React.createClass({
   },
 
   onPress: function() {
-    eventBridge.onPress({"name":"Play"});
+    eventBridge.onPress({"name":"play"});
+  },
+
+  onSocialButtonPress: function(buttonName) {
+    console.log(" in index on social button")
```

Comment:
Can we change this log to use Log.log, and have a more professional wording? like "Social Button Pressed: " + buttonName